src/utils/daos_hdlr.c

```diff
@@ -2818,8 +2818,10 @@ cont_clone_hdlr(struct cmd_args_s *ap)
 out:
 	D_FREE(src_str);
 	D_FREE(dst_str);
-	D_FREE(ca->src);
-	D_FREE(ca->dst);
+	if (ca) {
+		D_FREE(ca->src);
+		D_FREE(ca->dst);
+	}
```

Comment:
It looks like we also don't `D_FREE(ca)`? (alloc'ed at the beginning of this function).

Reply:
Looks like the memory of ca will be accessed later at,
https://github.com/daos-stack/daos/blob/master/src/control/cmd/daos/container.go#L905-L906
Maybe I can add code to free ap.dm_args (ca) in Go.

Reply:
I see... Anyway, not a blocker for this fix

Reply:
It looks like the memory is freed in Go code already. No worry about freeing ca here.
https://github.com/daos-stack/daos/blob/master/src/control/cmd/daos/util.go#L193-L195
	if ap.dm_args != nil {
		C.free_daos_alloc(unsafe.Pointer(ap.dm_args))
	}

Reply:
Thanks for checking on that!